openapi-cli/src/main/java/org/ballerinalang/openapi/cmd/OpenApiCmd.java

```diff
@@ -130,21 +133,28 @@ public void execute() {
                 Filter filter = new Filter(tag, operation);
                 try {
                     openApiToBallerina(fileName, filter);
+                    exitWithCode(0);
+                    return;
```

Comment:
Seems these last 4 returns this one and the rest seems not needed in this void method. You can safely remove those.